src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -36,15 +37,17 @@ internal class AnalyzerExecutor
         /// Optional delegate which is invoked when an analyzer throws an exception.
```

Comment:
Is the name `AnalyzerActionsExecutor` in the `<summary>` element above up-to-date? Consider using `cref` for it.